src/scf-release/src/acceptance-tests-brain/test-scripts/017_syslog_forwarding_test.rb

```diff
@@ -45,21 +45,23 @@ def show_env
     SCF_LOG_HOST = ENV.fetch('SCF_LOG_HOST', '')
     SCF_LOG_PORT = ENV.fetch('SCF_LOG_PORT', '514')
     SCF_LOG_PROTOCOL = ENV.fetch('SCF_LOG_PROTOCOL', 'tcp')
+
+    $KUBERNETES_NAMESPACE = ENV['KUBERNETES_NAMESPACE']
+
+    KUBERNETES_DOMAIN_SUFFIX = ".#{$KUBERNETES_NAMESPACE}.svc.#{ENV['KUBERNETES_CLUSTER_DOMAIN']}"
+
     if SCF_LOG_HOST.empty?
-        message = "SCF_LOG_HOST not set"
+        message = "SCF_LOG_HOST not set; expected to end with cluster domain (#{KUBERNETES_DOMAIN_SUFFIX})"
         STDERR.puts "\e[0;1;31m#{message}\e[0m"
         show_env
-        fail message
+        exit 99
```

Comment:
Is the change from `fail` to `exit 99` intended, and if yes, why ? (The new code feel less clear).

Reply:
Yes, because that's our magic code for `skip` (instead of `fail`). I should probably add a comment though?

Reply:
We have a command for skip in the testutils.

Reply:
Oh! Thanks! Force-pushed using that. (And dropped the second one, since if it's set we should probably fail instead of skip)